Exercises/objects/objects.js

```diff
@@ -0,0 +1,39 @@
+// Task #1
+const objectWithDefinedProperties = {};
+Object.defineProperties(objectWithDefinedProperties, {
+    eyeColor: { value: "blue", writable: false },
```

Comment:
When using `defineProperties` by default all property config params are false, not true